app/routes/routes.py

```diff
@@ -0,0 +1,119 @@
+from flask import Blueprint,jsonify, abort, make_response,request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from dotenv import load_dotenv
+
+
+
+goals_bp = Blueprint ("goals", __name__, url_prefix="/goals")
+
+
+def handle_id_request(cls,id):
+
+    try:
+        id = int(id)
+    except:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {id} invalid"}),400))
+    Object = cls.query.get(id)
+    if not Object:
+        abort(make_response({"message":f"{cls.__name__} {id} not found"},404))
+        
+    return Object
+
+
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+
+    request_body = request.get_json()
+    if "title" not in request_body:
+        return make_response({"details": "Invalid data"},400)
+    
+    new_goal = Goal.from_dict(request_body)
+    db.session.add(new_goal)
+    db.session.commit()
+    goal_response = new_goal.to_dict()
+
+    return make_response(jsonify ({"goal":goal_response}), 201)
+
+
+@goals_bp.route("", methods=[ "GET"])
+def read_all_goals():
+
+    sorting_query = request.args.get("sort")
+    if sorting_query == "asc":
+        goals=Goal.query.order_by(Goal.title.asc())
+    elif sorting_query=="desc":
+        goals=Goal.query.order_by(Goal.title.desc())
+    else:
+        goals = Goal.query.all()
```

Comment:
Nitpick: Overall, when you submitted your project, your code style is very inconsistent with itself about spaces. Try to normalize on one style, probably putting spaces around `=` and `==`:

```python
    if sorting_query == "asc":
        goals = Goal.query.order_by(Goal.title.asc())
    elif sorting_query == "desc":
        goals = Goal.query.order_by(Goal.title.desc())
    else:
        goals = Goal.query.all()
```